BookOfHabits/BookOfHabitsMicroservice.Domain.Entity/Propertys/Delay.cs

```diff
@@ -4,29 +4,33 @@ namespace BookOfHabitsMicroservice.Domain.Entity.Propertys
 {
     public class Delay: Property
     {
-        public Habit Habit { get; }
-        public bool IsAfterATime { get; }
-        public long AfterTime { get; }
-        public DateTime Start { get; }
-        public DateTime Finish { get; }
-        public bool IsEndless { get; }
-        public Delay(Guid id, Habit habit, bool isAfterATime, long afterTime, DateTime start, DateTime finish, bool isEndless) 
+        readonly int _ticksSec = 10_000_000;
```

Comment:
the fields int_ticksSec and int_day are had in TimeResetInterval model as well with the same values.

I have no clue about the logic, but it seem like code duplication. Because if I decide to change it, I will be forced to find all places with this variables. Instead of it I can determine constants and use it in the both models so it will be great solution to simplify  understanding code and make some cahnges in it in the long run.